src/css/flags/audioplayer.less

```diff
@@ -43,3 +43,9 @@
 .jwplayer.jw-flag-audio-player .jw-preview {
     display: none;
 }
+
+.jwplayer.jw-flag-media-audio.jw-flag-audio-player {
```

Comment:
That doesn't work in both cases. If we look above the stuff I added, we have this:

 .jwplayer.jw-flag-audio-player .jw-preview {
     display: none;
 }

But this doesn't work for when the player is at a small size.